example/config/spiffe-csi-driver.yaml

```diff
@@ -1,3 +1,117 @@
+---
+
+# Source: spire-agent/templates/csi-serviceaccount.yaml
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: spiffe-csi-driver
+  namespace: spire
+
+---
+
+# Source: spire-agent/templates/spiffe-csi-driver.yaml
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: spiffe-csi-driver
+  namespace: spire
+  labels:
+    app: spiffe-csi-driver
+spec:
+  selector:
+    matchLabels:
+      app: spiffe-csi-driver
```

Comment:
Is this needed if we don't have a matching `Service`?

Reply:
This spec is required to deploy the `DaemonSet`